apps/angular/src/app/anime-dashboard/anime-dashboard.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit } from '@angular/core';
+import { CommonModule, DatePipe } from '@angular/common';
+import { MatTableModule } from '@angular/material/table';
+
+import { map } from 'rxjs/operators';
+
+import { ApiService } from '.././services/api.service';
+import { AnimeFilm } from '../entities/animeFilm';
+
+/** Anime dashboard. */
+@Component({
+	selector: 'camp-anime-dashboard',
+	standalone: true,
+	imports: [CommonModule, MatTableModule, DatePipe],
+	templateUrl: './anime-dashboard.component.html',
+	styleUrl: './anime-dashboard.component.css',
+})
+export class AnimeDashboardComponent implements OnInit {
+
+	/** Displayed columns. */
+	public displayedColumns: string[] = ['imageSourceURL', 'titleEnglish', 'titleJapan', 'airedStart', 'filmType', 'filmStatus'];
+
+	/** Anime films. */
+	public animeFilms: AnimeFilm[] = [];
+
+	public constructor(private apiService: ApiService) {}
```

Comment:
Let's use a modern approach to inject dependencies.
```suggestion
	private readonly apiService = inject(ApiService);
```

Reply:
Also, it's bad practice to use such words like `Api, Http`, etc. inside our components. It would be better to rename this service to `AnimeService`, for example.